terraform/cloudfoundry/rds_broker.tf

```diff
@@ -37,3 +36,21 @@ resource "aws_security_group" "rds_broker_dbs" {
     Name = "${var.env}-rds-broker-dbs"
   }
 }
+
+resource "aws_db_parameter_group" "rds_broker_postgres95" {
+  name = "rdsbroker-postgres95-${var.env}"
+  family = "postgres9.5"
+  description = "RDS Broker Postgres 9.5 parameter group"
+
+  parameter {
+    apply_method = "pending-reboot"
```

Comment:
It might be useful to have some rationale in the commit message about why the `apply_method` is different for both parameters (although I have no context of db parameter groups so may be missing something)